### ISSUE TITLE
Missing specs with Bool type for models

### ISSUE THREAD

**Issue by jwoertink** [luckyframework/avram#288]
I was looking for an example in our spec models of a Bool, and was surprised to see we didn't have one.

```
[11:07AM] avram (features/239)$ ack "Bool" spec/support
spec/support/bucket.cr
3:    column bools : Array(Bool)
[11:07AM] avram (features/239)$
```

We should add one in  just to make sure we have that case covered.

**Comment by wout** [comment]
What do you propose? Would a few relevant specs on the `User` model suffice?

**Comment by jwoertink** [comment]
Just in this spec https://github.com/luckyframework/avram/blob/master/spec/model_spec.cr We should have one of the test models adding in a Bool column, and then testing that the model has a bool method on it. Nothing too crazy. It just helps if someone uses specs for documentation (like I do 😂 )

**Comment by wout** [comment]
Ok, I have got that ready. But this got me thinking, wouldn't we expect the getter for a Bool columns to have the `?` variant, just as we are used to having in Rails?

So for example, adding a `column available_for_hire : Bool` would create two getters:

```cr
user = User.new(available_for_hire: nil)
user.available_for_hire # => nil
user.available_for_hire? # => false
```

**Comment by paulcsmith** [comment]
@wout I never had a chance to add an issue for that but yes adding the ? method would be amazing!

**Comment by paulcsmith** [comment]
Closed by #300
